core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/version/VersionCreateRequest.java

```diff
@@ -195,10 +197,35 @@ public Boolean execute(RepositoryContext context) {
 				monitor.worked(1);
 //			});
 			
+			//Find resources that depend on this resource
+			List<String> dependentResourceIds = ResourceRequests.prepareSearch()
+				.setLimit(10_000)
+				.filterByDependency(String.format("(uri:%s OR uri:%s/* OR uri:%s\\?*) AND scope:domain) ", resource, resource, resource))
```

Comment:
Restricting to just the domain scope is an invalid assumption. This code should work regardless of the scope value of the dependency entries.